File: solutions/080.remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int len = nums.length;
        if(len <=2 ) return len;
        int counter = 2;
        for(int i = 2; i < len; i++){
            if(nums[counter - 2] != nums[i]){
                nums[counter] = nums[i];
                counter++;
            }
        }
        return counter;
    }
}

File: solutions/080.remove-duplicates-from-sorted-array-ii/remove-duplicates-from-sorted-array-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    let index = 0;
    for(let i = 0; i < nums.length; i++){
        if(index < 2){
            nums[index++] = nums[i];
        }else{
            if(nums[i] !== nums[index - 2]){
                nums[index++] = nums[i];
            }
        }
    }
    return index;
    
};